app/models/submission.rb

```diff
@@ -17,6 +17,8 @@ class Submission < ActiveRecord::Base
   validates :english_keywords, presence: true
   has_many :authorships, dependent: :destroy
   has_many :article_revisions, dependent: :destroy
+  #zgłoszenie ma wiele powiązanych ze sobą wiadomości
+  has_many :submission_mails, dependent: :destroy
```

Comment:
W sumie to będzie powodowało pewne problemy. Proszę zmienić nazwę na `messages`.